core/modules/filter/filter.pages.inc

```diff
@@ -383,7 +262,7 @@ function filter_format_editor_link_form($form, &$form_state, $format) {
     ),
     '#weight' => -10,
     '#maxlength' => 256,
-    '#autocomplete_path' => 'filter-link/autocomplete',
+    '#autocomplete_path' => 'path/autocomplete',
```

Comment:
could this be just `path-autocomplete`? It isn't part of a `path` module and is in generic use, so no real need for fragments.

Reply:
If I do a search of core for `autocomplete_path` I see other ones such as `taxonomy/autocomplete`, `user/autocomplete`, `node/autocomplete`, etc. So that's why I did it this way. There's no other uses of `*-autocomplete` in core, so this'd be the first if we did it that way... I'm not opposed to it necessarily, but just explaining my reasons. Also, I consider 'path' to be generic, not specific to a module/piece-of-functionality.

Reply:
Sure, will do.